core/pom.xml

```diff
@@ -22,6 +23,14 @@
     <commons-gwt.version>10.2.0.0-SNAPSHOT</commons-gwt.version>
     <commons-xul.version>10.2.0.0-SNAPSHOT</commons-xul.version>
   </properties>
+  <dependencies>
```

Comment:
Are we actually using the Junit5 API?  If not we probably shouldn't bother importing it yet.